magic/divine/healmagic.cpp

```diff
@@ -127,17 +127,37 @@ int getHeal(const std::shared_ptr<Creature>&healer, std::shared_ptr<Creature> ta
 
         switch(healer->getDeity()) {
         case CERIS:
-            if(healer->getAdjustedAlignment() == BLOODRED || healer->getAdjustedAlignment() == ROYALBLUE) {
-                healer->print("Your healing is not as effective while so far out of natural balance.\n");
+            if(healer->getAdjustedAlignment() <= REDDISH) {
+                healer->print("Your healing is not as effective with such darkness in your heart.\n");
                 level /= 2;
             }
 
-
             mod = level / 2 + Random::get(1, 1 + level / 2);
-            if(healer->getAdjustedAlignment() == NEUTRAL) {
+            if(healer->getAdjustedAlignment() == ROYALBLUE) {
```

Comment:
Ceris has been a neutral class so far, is that changing?

Reply:
yes. Code adjusted elsewhere now to reflect.